client/src/components/Atoms/Wrapper.tsx

```diff
@@ -11,7 +11,13 @@ export const Inner = styled.div`
   margin-right: auto;
   margin-left: auto;
   width: 1020px;
-  min-width: 1020px;
+  padding: 0 30px;
+`
```

Comment:
I was playing around with this a bit and figured out that you can just change Inner to these styles and everything seems to work:
```ts
export const Inner = styled.div`
  display: flex;
  margin-right: auto;
  margin-left: auto;
  width: 100%;
  max-width: 1020px;
  padding: 0 30px;
`
```

an advantage here is that the main panel width for the audit board view will then match the width of the Header, which also uses Inner